section3/reflection.md

```diff
@@ -2,12 +2,33 @@
 
 1. In Section 3, we removed even more supports and pushed you to learn a bit more independently, read through technical documentation, and use your resources more. How did that feel?
 
+Overall it was pretty fun. I enjoy the hunt for clues and solving the puzzle. I reached out for
+help on Slack for the first time too, which I knew I would have to do eventually and it was good
+to rip off that bandaid.
+
 2. Think of a web or mobile application you use regularly. Where is an example of a place that the developers probably use Arrays? Explain.
+An example could be a playlist in Tidal. The elements of the array are songs. By rearranging
+the songs in the playlist you would change the index positions.
 
 3. In your own words, explain what an Array method is.
+An array method is a tool in JS that performs a specific function. They can allow you
+to update/change the array without having to go back into array itself. You cannot
+change what an array method does.
 
 4. What is the purpose of a loop? When we would use a loop in conjunction with array vs. without?
+A loop is an efficient way to tell your code to perform an action however many times you want
+without having to write code for each individual action. Using a loop in conjunction with
+an array allows you to ask your code to perform an action on multiples pieces of data, in this
+case all of the elements in your array. You would use a loop not in conjunction with an array
+if you did not want or need that array to be looped through.
 
 5. If you had to teach a 5 year old about loops, how would you explain it?
+A loop is like a magical machine that will say what you want it to say however
+many times you want to say it. So lets say your favorite food is pancakes, and you want your computer
+to say "I Love Pancakes!" a hundred times. You could type that sentence out, over and over, but that
+would be a lot of work. Instead you could use a few magic words to build your magic loop machine,
+tell it that you want it to say "I Love Pancakes!" a hundred times and then sit back and watch your
+computer say back how much you love pancakes.
 
 6. What questions do you still have about Arrays and loops?
+Does the order of conditional statements affect how the loop runs, even if it's only going through one array? Is it possible to overexplain your conditions, in a way that is not only inefficient but affects what prints back? 
```

Comment:
Yes and yes! Order _can_ matter, and it's especially common to unintentional write our if/else if/else statements with branches that will never execute when we use the OR (`||`) operator. You'll get lots of logic practice on your projects!